Concurrency.xcodeproj/xcshareddata/xcschemes/Concurrency-Package.xcscheme

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need to check this file in ?

Reply:
As discussed offline, we'll gitignore this, since it is generated by SPM.